src/energiapy/model/constraints/cost.py

```diff
@@ -63,19 +63,16 @@ def constraint_transport_imp_cost(instance: ConcreteModel, scheduling_scale_leve
     Returns:
         Constraint: transport_imp_cost
     """
-    scales = scale_list(instance=instance,
-                        scale_levels=scheduling_scale_level+1)
 
     def transport_imp_cost_rule(instance, sink, source, resource, transport, *scale_list):
         if trans_cost is None:
```

Comment:
guard clause design pattern might make this more readable

Reply:
Gotcha